src/query/sql/src/planner/binder/aggregate.rs

```diff
@@ -74,6 +78,11 @@ pub struct AggregateInfo {
     /// TODO(leiysky): so far we are using `Debug` string of `Scalar` as identifier,
     /// maybe a more reasonable way is needed
     pub group_items_map: HashMap<String, usize>,
+
+    /// Index for virtual column `grouping_id`. It's valid only if `grouping_sets` is not empty.
+    pub grouping_id_index: IndexType,
```

Comment:
Better to be `Option<IndexType>`? If `grouping_sets` is empty, it's `None`

Reply:
Option<IndexType> will consume more memory.

![image](https://user-images.githubusercontent.com/33975039/225192480-86722d51-cfbd-4e63-9ca6-6f1ae70e160f.png)